io/io/src/TFile.cxx

```diff
@@ -1333,7 +1333,7 @@ TList *TFile::GetStreamerInfoList()
          return 0;
       }
       key->ReadKeyBuffer(buf);
-      list = dynamic_cast<TList*>(key->ReadObjWithBuffer(buffer));
+      list = dynamic_cast<TList*>(key->ReadObjWithBuffer(buffer,0,1));
```

Comment:
if the argument of the function are declared boolean, please use kTRUE and kFALSE.